### DISCUSSION TITLE
What GO Setup should I use for this converters and cables mess?

### DISCUSSION THREAD

**Discussion by cmsotelo** [GrandOrgue/grandorgue#2013] [Help]
Hi!
I want to use GO in a little parish, I'm trying to use what I already have

My setup is like this:
GO in Laptop stereo output ----> Converter Jack 3.5 to 6.35 (stereo I/O) 
<img src="https://github.com/user-attachments/assets/e99d6c57-52f6-426f-bede-ae0b705ccf70" width=200>
---> Jack mono cable ---> Jack to XLR Conv. 
<img src="https://github.com/user-attachments/assets/7f2f1e65-afa6-4631-aa6a-d51073863b85" width=200>
 ----> XLR Cable ---> XLR Input Mixer ----> 2 Speakers.

after this mess I think my real input will be mono, right?
What should be my GrandOrgue Audio Setup for this? 

Does it is the same use 1 channel with L+R group than 2 channels with L & R separately.

Should I load the samples in Mono/Stereo or it doesn't matter?

**Comment by midimusic** [comment]
You should be aiming for stereo. Thus GO should be set up for stereo.
Your adaptors and cables should all be stereo. Thus use of your "jack mono cable" is wrong.
csw900

**Reply by cmsotelo** [reply]
> You should be aiming for stereo. Thus GO should be set up for stereo. Your adaptors and cables should all be stereo. Thus use of your "jack mono cable" is wrong. csw900

I forgot to mention i'm doing all this because i'm far from the mixer.

If i should go for stereo then I would need a cable/adapter from the jack to a 2 xlr (xlr sends mono signals, it doesn't?)

and the other question about the setup config in grandorgue? (samples loaded in mono/stereo and config in the Audio tab in Settings) In mono signal are the same?

**Reply by larspalo** [reply]
> If i should go for stereo then I would need a cable/adapter from the jack to a 2 xlr (xlr sends mono signals, it doesn't?)

Yes, you usually would need two input channels on the mixer panned hard L/R to transfer the stereo signal if (two normal) XLR cables are used. But many mixers also has dedicated input channels for stereo inputs by jack and/or RCA that keep the real stereo signal. If you need to send the signal with XLR then some kind of DI box can possibly be useful (like https://www.thomann.de/se/radial_engineering_pro_av2.htm or anything similar) that allow easy conversions. But there are other (cheaper) options too like https://www.thomann.de/se/cordial_cfy_1.5_wmm_long.htm.

> In mono signal are the same?

If mono loading is selected in GO, or the samples actually are in mono, the left (single) channel will be duplicated to the right channel output for that audio group.

**Reply by cmsotelo** [reply]
Ok, so by rca you mean the red and white inputs from here...
![1](https://github.com/user-attachments/assets/3d2cadff-9f06-4373-af7e-a1a25a49a777)

and idk if this hi-z inputs are stereo honestly...
![2](https://github.com/user-attachments/assets/9bbef483-de67-4993-9304-635a29623242)

That would be my only two options I guess.

**Reply by larspalo** [reply]
> and idk if this hi-z inputs are stereo honestly...

No, it usually wouldn't be. Hi-Z just refers to the impedence level that an instrument like guitar would have (as opposed to the low-z of the XLR mic inputs).

Exactly what the RCA inputs above go to isn't possible to see from the photo, but it seems that you have potentiometers on the front for both cd/tape and mp3 inputs - I'd bet on the cd/tape one but you'll have to test it for yourself.

If you cannot get at least two of the mic mixer channels free you'll need to use the RCA inputs instead. If there's a very substantial distance for the cable length there might be issues, but it should be possible to deal with. As far as I can google, 100 feet cable length should be quite ok and without any signal loss.

**Reply by cmsotelo** [reply]
I connected the rca cables and works perfectly BUT, sadly I couldn´t aprreciate any difference from the mono input I did before. And that was (apparently) because I don't have a pair of stereo speakers but 2 Mono outputs :((((((
I tried to pan the volume for mute one channel and both speakers sounds always, after that I checked the mixer:
![3](https://github.com/user-attachments/assets/b30892d2-092f-4c9e-9425-f251fa63839a)

Speaker Output dont say Left Right or anything, so they are both mono :((((

**Comment by midimusic** [comment]
You really ought to determine (and tell us) the make and model of your mixer/amplifier and find its proper specification. From the pictures It looks as if it is a stereo device. You need two channels for stereo all the way from GO to the speakers. You can easily test this by disconnecting each input channel in turn and checking that only ONE speaker outputs sound for each channel.

You should be using the "line out" from your computer (or headphone out) and connecting it to the CD input sockets (i.e. RCA connectors). You should not be using microphone inputs. Bear in mind that your computer output may not be stereo even if GO is set up for stereo.

csw900

**Reply by cmsotelo** [reply]
I think it is just a generic cheap and low quality mixer. I couldn't find this "Roncosound" specific in google but I found others very similar and in a forum (in spanish, sorry!! https://foros.doctorproaudio.com/showthread.php?21684-Ayuda-Consola-Potenciada/page2) saying its a 2 mono outputs mixer.

![5](https://github.com/user-attachments/assets/1521af65-e486-4796-81a3-51c8b291eebc)

I connected the pc from the headphones output to the rca of the mixer of course. 
I tried panning with pavucontrol the L/R signals to test if there were stereo but both sides played the same, so thats all, sadly.

**Reply by midimusic** [reply]
It is now pretty clear that it is a mono mixer/amplifier. I would bin it and buy a proper stereo amp.

From the above comments GO will load only the left hand channel if it is in mono mode. This will not produce a proper mono output. The proper way to get mono from stereo is to ADD the two channels. You can probably do this by loading GO in stereo mode and wiring the two stereo channels together which is what it looks like your "jack mono cable" is doing.

csw900
